doc/pages/About.md

```diff
@@ -1,60 +1,62 @@
+`aspcud` external solver, if available.
+
+At the end of 2012, Vincent Bernardoff and Guillem Rieu (from OCamlPro) worked for a few months on improving the documentation and ease of use of opam for newcomers. They created [opam2web](https://github.com/ocaml/opam2web), the tool used to generate https://opam.ocaml.org.
+
+The [first public beta of opam](http://www.ocamlpro.com/blog/2013/01/17/opam-beta.html)
+ was released in Jan 2013, and a few
+months later (in March 2013), [the first official release landed](http://www.ocamlpro.com/blog/2013/03/14/opam-1.0.0.html).
+A few days later, Louis Gesbert (who joined OCamlPro in Dec 2012)
 pushed [his first commit](https://github.com/ocaml/opam/commit/c56cf5e1e244cee9f707da8b682996bbc5dd31ff)
 to the codebase. In Nov 2013,
-[OPAM 1.1.0](https://opam.ocaml.org/blog/opam-1-1-0-released/) was released and Louis became the
+[opam 1.1.0](https://opam.ocaml.org/blog/opam-1-1-0-released/) was released and Louis became the
 technical lead. A months later, [opam 1.1.1](https://opam.ocaml.org/blog/opam-1-1-1-released/) with numerous bug fixes.
 
-### opam-repository
+### `opam-repository``
 
-Meanwhile, in June 2012 Mirage started to use opam (as it was using a
-custom 3.12.1 compiler). Very quickly, starting off Frederic's work, Anil and Thomas shared the task
-of adding new packages and monitor the pull-requests on
-opam-repository. The initial policy was to accept as many packages as
+Meanwhile, in June 2012, MirageOS started to use opam (as it was using a
+custom 3.12.1 compiler). Very quickly, building on Frederic's work, Anil and Thomas shared the task
+of adding new packages and monitored the pull requests on
+`opam-repository`. The initial policy was to accept as many packages as
 possible, which means that things were often broken. So they started
 to use background [bulks builds](https://github.com/avsm/opam-bulk-logs)
  to improve the overall repository
 quality. In July 2012, Jane-Street's Core libraries made [its
 apparition](https://github.com/ocaml/opam-repository/commit/bad688d0f49f6c750525b0047b336eb8606e419d)
  in the repository. To improve the quality of new
-packages, Travis CI was [integrated](https://github.com/ocaml/opam-repository/commit/2671cb1e968e084c13989762ea43fc1a5b4703d7) in Sept 2013 to the pull-request
-process. From Aug to Nov 2013, all the contributors of opam-repository
-were contacted to re-license their contribution to CC0, which enable the
+packages, Travis CI was [integrated](https://github.com/ocaml/opam-repository/commit/2671cb1e968e084c13989762ea43fc1a5b4703d7) in Sept 2013 to the pull request
+process. From Aug to Nov 2013, all the contributors of `opam-repository`
+were contacted to relicense their contribution to CC0, which enabled the
 [move of the repository](https://github.com/ocaml/opam-repository/issues/955)
-to the `ocaml` organisation. The [opam
+to the OCaml organisation. The [opam
```

Comment:
```suggestion
to the `ocaml` organisation. The [opam
```

Reply:
Are you specifically talking about the `ocaml` command here?